v2/internal/attacktechniques/entra-id/persistence/hidden-au/main.go

```diff
@@ -0,0 +1,201 @@
+package entra_id
+
+import (
+	"context"
+	_ "embed"
+	"errors"
+	"fmt"
+	"github.com/aws/smithy-go/ptr"
+	"github.com/datadog/stratus-red-team/v2/pkg/stratus"
+	"github.com/datadog/stratus-red-team/v2/pkg/stratus/mitreattack"
+	graphmodels "github.com/microsoftgraph/msgraph-sdk-go/models"
+	"log"
+	"strings"
+)
+
+//go:embed main.tf
+var tf []byte
+
+func init() {
+	stratus.GetRegistry().RegisterAttackTechnique(&stratus.AttackTechnique{
+		ID:           "entra-id.persistence.hidden-au",
+		FriendlyName: "Create Hidden Scoped Role Assignment Through HiddenMembership AU",
+		Description: `
+Creates an [Administrative Unit (AU)](https://learn.microsoft.com/en-us/graph/api/resources/administrativeunit?view=graph-rest-1.0) with hidden membership, and a scoped role assignment over this AU.
+This simulates an attacker that TODO.
```

Comment:
@siigil I think this requires a few sentences to explain the scenario (which is quite complex)